deepchecks/core/suite.py

```diff
@@ -140,73 +136,14 @@ def widget_serializer(self) -> SuiteResultWidgetSerializer:
         """Return WidgetSerializer instance."""
         return SuiteResultWidgetSerializer(self)
 
-    @property
-    def ipython_serializer(self) -> SuiteResultIPythonSerializer:
-        """Return IPythonSerializer instance."""
-        return SuiteResultIPythonSerializer(self)
-
     @property
     def html_serializer(self) -> SuiteResultHtmlSerializer:
         """Return HtmlSerializer instance."""
         return SuiteResultHtmlSerializer(self)
 
-    def show(
```

Comment:
we don't have show anymore?

Reply:
have, but it comes from `DisplayableResult` class